docs/hub/_toctree.yml

```
@@ -243,6 +243,8 @@
       title: How to Add a Space to ArXiv
     - local: spaces-cookie-limitations
       title: Cookie limitations in Spaces
+    - local: spaces-url-query-hash
```

```suggestion
    - local: spaces-handle-url-parameters
```
maybe a clearer URL for this doc?